CoreWidgetProvider/CoreWidgetProvider.csproj

```diff
@@ -66,4 +69,6 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+
+  <ProjectExtensions><VisualStudio><UserProperties widgets_4templates_4sshwalletpatternmatching_1json__JsonSchema="https://codemagic.io/codemagic-schema.json" /></VisualStudio></ProjectExtensions>
```

Comment:
What is this?

Reply:
I honestly don't know - I didn't see that in the git diff. I didn't manually put that there.

Reply:
Removed it and project builds as expected.